test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -1670,6 +1670,98 @@ hw.module @IndexPartSelect(out a : i3) {
   hw.output %c : i3
 }
 
+// Functions.
+sv.func private @function_declare1(in %in_0 : i2, out out_0: i2, in %in_1 : i2, out out_1 : i1)
+sv.func private @function_declare2(in %in_0 : i2, in %in_1 : i2, out out_0 : i1 {sv.func.explicitly_returned})
+sv.func private @function_declare3(in %in_0 : i2, out out_0 : i2, out out_1 : i1 {sv.func.explicitly_returned})
+
+// CHECK-LABEL: func_call
+hw.module @func_call(in %in_0 : i2, in %in_1 : i2, in %in_2: i1, out out: i1) {
+  %fd = hw.constant 0x80000002 : i32
+  // CHECK:      wire        _function_declare2_0;
+  // CHECK-NEXT: logic [1:0] _function_declare3_0;
+  // CHECK-NEXT: logic       _function_declare3_1;
+  // CHECK-NEXT: logic [1:0] _function_declare1_0;
+  // CHECK-NEXT: logic       _function_declare1_1;
+  // CHECK-NEXT: _function_declare2_0 = function_declare2(in_0, in_1);
```

Comment:
I think this has to be
```
assign _function_declare2_0 = function_declare2(in_0, in_1);
```

Reply:
Fixed by https://github.com/llvm/circt/pull/7015/commits/68a2872706733514725beefdcd172300a53a1559. Thank you for finding this!